### ISSUE TITLE
Sensors stop working once in a while

### ISSUE THREAD

**Issue by yfaber** [kevinvincent/ha-wyzesense#157]
Every other week HA won't poll the info from the sensors, i have to turn off HA and turn it on again... any ideas on what it's going on?

**Comment by VarenDerpsAround** [comment]
I hate this, I hate this so much. Boost this please. @photinus ? I get this same situation but seeing as I have 20 sensors running, it's a fair bit shorter for me personally. So so so many people I see have this issue. I have to completely redo mine every like week it seems. It will just stick on one status, and not move, I have to cold boot to get them back working, which needless to say, with this particular integration, and proxmox, causes issues all of it's own. Bro. How can I help? Help me learn this python app, I'll do anything in my power to help me, help you, help us. Please. I'll pay you! What's your buy-me-a-coffee, you make this work right, you got quite a few people I know who would drop you a line. I've been boosting wyze to get off their asses and make their own integration, well we are still stuck with this so far, so hopefully, we can get it stable. I'd love to be able to confidently run my security system, once again, as the integration worked fine for several months, then a couple updates flew by and now it's busted af. So please, how can I help you pho? I'll learn the entirety of python if I have to. 

Happy belated 4th!

**Comment by RoldyBuffalo** [comment]
More Crumbs

```
Log Details (ERROR)
Logger: custom_components.wyzesense.wyzesense_custom
Source: custom_components/wyzesense/wyzesense_custom.py:364
Integration: wyzesense (documentation)
First occurred: July 7, 2020, 10:30:36 AM (8 occurrences)
Last logged: 4:56:17 PM

Mismatched checksum, remote=0763, local=0773
Invalid packet: b'55aa532319000001732fa5380aab373739303035354602040700000001000002000101370005ca'
Mismatched checksum, remote=05CA, local=05C3
Invalid packet: b'55aa'
Invalid packet length: 2
```

**Comment by kevinvincent** [comment]
Can you try the latest release and see if you still see it as often?

**Comment by javierdc** [comment]
I'm using the latest version (0.0.9) and still see the issue.

Thanks - Javier

**Comment by kevinvincent** [comment]
Does it stop working after you get those error messages? Or does it log those errors but still work?

**Comment by phreaq** [comment]
> Does it stop working after you get those error messages? Or does it log those errors but still work?

I am having the same issue as the other posters. I am using 0.0.9, and my sensors (6 different ones) will stop updating their status. all of them will loose the ability at the same time, not just a sub-set of them. they do not come back online until I reboot HA

**Comment by Phara0h** [comment]
Even after 0.9 still have the same issues as before

**Comment by RoldyBuffalo** [comment]
Still, still, still have the same issues as before. Appreciate the update after 3 months of being stagnant, but it just broke it even more.

**Comment by RoldyBuffalo** [comment]
@kevinvincent yes, we're all still having the same issues as before, I personally am having the issue stated 10x as often now. it won't even stay online for 10 minutes. Reverted back to 0.0.8 for now, but this is kinda ridiculous.

**Comment by RoldyBuffalo** [comment]
This is interesting. It seems that when you set a initial state, it disables the sensor entirely. I just commented out all my initial states in my config and rebooted, and sensors started responding that were previously assumed to have been dead, weird. I was going through my sensors, thought this particular one was dead, but after removing the initial state in the config for binary sensors, it started working again? (previously to 0.0.9~ish, it wasn't doing this, maybe it was a hass update? maybe it was the sense update. but initial state used to just set the sensor to the no-motion/closed position on boot up.)

**Comment by phreaq** [comment]
> ...I personally am having the issue stated 10x as often now. 

How are you running HA? Is it on a Pi, VPC, NC? etc.?

**Comment by ideal2545** [comment]
Just wanted to comment that this integration has been working great for me but I do have to do a reboot of the entire HA Host OS about weekly right now to get the sensors to come back online. It was working very well for nearly 4-6 months before this started happening, nothing too crazy since its just a reboot but if you need any thing from me please let me know. I use ESXI and do passthrough.

**Comment by mattyb8562** [comment]
By chance are you guys running HA on Linux>Docker>Home Assistant Supervised (Hassio)? The reason I ask is, I was having a similar issue where my sensors would randomly stop responding. I would need to remove the Wyze bridge, restart ha, and insert the bridge before it boots to bring them all back. Recently I installed a Z-Wave stick and it had this bit in the documents about disabling Modem Manager in ubuntu. It warned about zwave node random disconnects, unresponsive etc. Since I have disabled Modem Manager I am no longer having the issue with wyze. I'm wondering if the same issue applies to the wyze bridge as it does with zwave usb sticks.

**Comment by Phara0h** [comment]
It starts working again after a good old fashion reboot

**Comment by yfaber** [comment]
> It starts working again after a good old fashion reboot

Is there a way to reboot it automatically ?

**Comment by cheechie** [comment]
Just stop using these sensors and move to zigbee.   I tried everything possible with these wyze sensors but they were never reliable.  I have been using sonoff zigbee without issue for over a month.

**Comment by yfaber** [comment]
> Just stop using these sensors and move to zigbee. I tried everything possible with these wyze sensors but they were never reliable. I have been using sonoff zigbee without issue for over a month.

Can u share the URL for the sensors?

**Comment by cheechie** [comment]
> > Just stop using these sensors and move to zigbee. I tried everything possible with these wyze sensors but they were never reliable. I have been using sonoff zigbee without issue for over a month.
> 
> Can u share the URL for the sensors?

If you have time to wait you can get it on AliExpress much cheaper. This is the bridge I used https://www.amazon.com/ZBBridge-Dual-protocol-Supporting-Multi-device-Management/dp/B08BFT44QL and there's a number of different sensors that work with this. I didn't even flash it with tazmotta as there's a home assistant component that integrates with sonoff ewelink.  

There are various bridges and sensors other than sonoff that work as well.
